package/openmrs-config/configuration/locations/locations.csv

```diff
@@ -0,0 +1,7 @@
+Uuid,Void/Retire,Name,Description,Parent,Tags,Tag|Facility Location,Attribute|9eca4f4e-707f-4bb8-8289-2f9b6e93803c,Attribute|Last Audit Date,Address 1,Address 2,Address 3,Address 4,Address 5,Address 6,City/Village,County/District,State/Province,Postal Code,Country,_order:1000
+8d6c993e-c2cc-11de-8d13-0010c6dffd0f,,Unknown Location,Unknown Location,,,,,,,,,,,,,,,,,
+44c3efb0-2583-4c80-a79e-1f756a03c0a1,,Outpatient Clinic,Outpatient Clinic,,Login Location,TRUE,,,,,,,,,,,,,,
+ba685651-ed3b-4e63-9b35-78893060758a,,Inpatient Ward,Inpatient Ward,,Login Location,,,,,,,,,,,,,,,
+8d9045ad-50f0-45b8-93c8-3ed4bce19dbf,,Mobile Clinic,Mobile Clinic,,Login Location,,,,,,,,,,,,,,,
+28066518-a0e5-4331-b23e-7b40f96d733a,,MCH Clinic,MCH clinic,,Login Location,,,,,,,,,,,,,,,
```

Comment:
What's MCH? Sounds like use-case specific, maybe we can remove it to keep the config as generic as possible.